### PR TITLE
Add support for `google_compute_router` to TGC

### PR DESCRIPTION
**Pull request by delimaneto**
adding support for compute.googleapis.com/Router
```release-note:none
```

### CODE REVIEW

**Review by melinath — COMMENTED** [review]
on hold pending discussions with service team

**Comment by sushant00** [comment]
@melinath can you please review this change now since internal discussion is closed.

**Comment by melinath** [comment]
yes, thanks for flagging!

**Review by melinath — CHANGES_REQUESTED** [review]
@delimaneto please resolve the merge conflicts

**Review by melinath — CHANGES_REQUESTED** [review]
please resolve merge conflicts

**Comment by delimaneto** [comment]
conflicts fixed

**Review by melinath — CHANGES_REQUESTED** [review]
LGTM apart from one stray change.

File: mmv1/templates/tgc/resource_converters.go.erb [hunk 1]
```diff
@@ -45,8 +45,9 @@ func ResourceConverters() map[string][]cai.ResourceConverter {
 		"google_compute_global_forwarding_rule":           {compute.ResourceConverterComputeGlobalForwardingRule()},
 		"google_compute_instance":                         {compute.ResourceConverterComputeInstance()},
 		"google_compute_network":                          {compute.ResourceConverterComputeNetwork()},
-		"google_compute_node_template": 				           {compute.ResourceConverterComputeNodeTemplate()},
 		"google_compute_route":                  		       {compute.ResourceConverterComputeRoute()},
+		"google_compute_router":						               {compute.ResourceConverterComputeRouter()},
+		"google_compute_node_template": 				           {compute.ResourceConverterComputeNodeTemplate()},
```
[the remark this answers is not in the document]
Comment (delimaneto) [reply mmv1/templates/tgc/resource_converters.go.erb]: I  did the undo and aligned all the resources that was with tab space

**Review by melinath — APPROVED** [review]
okay - approving this with the whitespace changes

**Pull request merged**